src/views/projectExplorer/migrateSource.ts

```diff
@@ -187,6 +193,36 @@ export async function migrateSource(iProject: IProject, library: string): Promis
     }
 }
 
+function fixExtensions(workspaceFolder: string): void {
+    const filesAndDirs = fs.readdirSync(workspaceFolder);
+
+    filesAndDirs.forEach((fileDir: string) => {
+        const path = `${workspaceFolder}/${fileDir}`;
+        const stats = fs.statSync(path);
+
+        if (stats.isDirectory()) {
+            fixExtensions(path);
+        } else {
+            const fileSplit = fileDir.split('.');
+            const extension = fileSplit.at(-1);
+
+            if (fileSplit.length === 3 && extension?.toUpperCase() === extension) {
+                fileSplit[1] = fileSplit[1].toUpperCase();
+
+                const newFileName = fileSplit.join('.');
+
+                fs.rename(path, workspaceFolder + "/" + newFileName, (error) => {
+                    if (error) {
+                        console.error(`Error fixing extension's case for file: ${fileDir}`);
+                    } else {
+                        console.log(`File ${fileDir} renamed successfully to ${newFileName}`);
```

Comment:
This is really an internal issue that we didn't get the case right the first time, so  I don't think we should issue a consule statement here.
In fact, couldn't this be fixed at the time we do the initial rename?  won't the  console log have the wrong case there?